modules/rand/abci.go

```diff
@@ -4,20 +4,21 @@ import (
 	"encoding/hex"
 	"fmt"
 
-	sdk "github.com/irisnet/irishub/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/irisnet/irishub/modules/rand/internal/types"
```

Comment:
use alias instead of import "internal/types" and "internal/keeper" in "rand" package